### DISCUSSION TITLE
[LG C1 55] Does this actually reduce OLED brightness or it's like a Dim light filter?

### DISCUSSION THREAD

**Discussion by bogas04** [MonitorControl/MonitorControl#1588] [Q&A] [answered]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Question

Just curious how this is working. Does it fall back to a software brightness fix in case it isn't possible over HDMI? Is there a way to know which mode it may be operating in? 

I ask this because the quality of greys looks a bit different and while I am able to reduce brightness I am not able to change volume, which makes me wonder if it's using CEC kind of tech or not.

**Comment by waydabber** [comment, marked as answer]
The LG C1 TV does not support DDC so software brightness used (the app changes the gamma table to adjust brightness).
